client/api.go

```diff
@@ -35,119 +37,99 @@ import (
 	"github.com/stripe/stripe-go/reversal"
 	"github.com/stripe/stripe-go/sku"
 	"github.com/stripe/stripe-go/source"
+	"github.com/stripe/stripe-go/sourcetransaction"
 	"github.com/stripe/stripe-go/sub"
 	"github.com/stripe/stripe-go/subitem"
 	"github.com/stripe/stripe-go/token"
+	"github.com/stripe/stripe-go/topup"
 	"github.com/stripe/stripe-go/transfer"
+	"github.com/stripe/stripe-go/usagerecord"
 )
 
 // API is the Stripe client. It contains all the different resources available.
 type API struct {
+	// Account is the client used to invoke /accounts APIs.
+	Account *account.Client
```

Comment:
Do you know why it says `Account` and not `Accounts`? Should we fix it?

Reply:
Hm, I'd guess because maybe the original package started out by just doing `GET /account`, which was singular.

Yeah, I think we should fix it, but maybe not now since it's breaking — maybe add a TODO or something here?

Reply:
Sounds good. I think we can punt on the TODO and I'll change it when we go ahead and remove the old 3DS endpoint